src/utils.hpp

```diff
@@ -45,28 +45,8 @@ struct eval_result {
     bool ephemeral;
 };
 
-struct object_limits {
-    static constexpr uint32_t default_max_container_depth{DDWAF_MAX_CONTAINER_DEPTH};
-    static constexpr uint32_t default_max_container_size{DDWAF_MAX_CONTAINER_SIZE};
-    static constexpr uint32_t default_max_string_length{DDWAF_MAX_STRING_LENGTH};
-
-    // Non-overridable limits
-    static constexpr uint32_t max_transformers_per_address{10};
-    static constexpr uint32_t max_key_path_depth{DDWAF_MAX_CONTAINER_DEPTH};
-
-    // User provided limits
-    uint32_t max_container_depth{DDWAF_MAX_CONTAINER_DEPTH};
-    uint32_t max_container_size{DDWAF_MAX_CONTAINER_SIZE};
-    uint32_t max_string_length{DDWAF_MAX_STRING_LENGTH};
-};
-
-inline size_t find_string_cutoff(const char *str, size_t length, object_limits limits = {})
+inline size_t find_string_cutoff(const char *str, size_t length)
```

Comment:
is this still needed?

Reply:
It's not, I have to convert it into documentation since the caller will have to take make sure to truncate strings correctly.

Reply:
Addressed in https://github.com/DataDog/libddwaf/pull/387/commits/95ba3b1f3430ba12ec3061a583669015d0d98deb